database/migrations/2022_11_28_200508_genre_medium.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('genre_medium', function (Blueprint $table){
+            $table->id();
+            $table->unsignedBigInteger('media_id');
+            $table->unsignedBigInteger('genre_id');
+            $table->timestamps();
+
+            $table->foreign('media_id')->references('id')->on('medium')->onDelete('cascade');
```

Comment:
should be  $table->foreign('medium_id')->references('id')->on('media')->onDelete('cascade');